CMakeLists.txt

```
@@ -5,6 +5,8 @@ endif()
 cmake_minimum_required(VERSION 3.5.1 FATAL_ERROR)
 project(MAPBOX_BASE LANGUAGES CXX C)
 
+option(FORCE_BUILD_TESTING "Bypass project target check and enforce building tests" OFF)
```

```suggestion
option(MAPBOX_BASE_FORCE_BUILD_TESTING "Bypass project target check and enforce building tests" OFF)
```
To avoid names conflict with other projects